packages/gatsby-theme/gatsby-node.js

```diff
@@ -33,26 +71,43 @@ exports.createPages = async ({ graphql, actions }) => {
     const tutorialPath = getTutorialSlug(node.fileAbsolutePath);
     const overviewPageSlug = getTutorialOverviewSlug(node.fileAbsolutePath);
     const overviewTemplate = require.resolve(
-      "./src/components/templates/TutorialOverview.tsx"
+      './src/components/templates/TutorialOverview.tsx',
```

Comment:
Let's go ahead and run prettier on all the files in the repo and I'll add a script to auto-format the code before committing so we don't have updates like these.